docs/host-list.rst

```diff
@@ -2,90 +2,150 @@
 
 .. _host-list:
 
-=============
-VPS providers
-=============
+===================
+Hosting Information
+===================
 
 .. note::
     This doc is written for the :ref:`hosting section <getting-started-hosting>`
     of the :ref:`getting started <getting-started>` guide. Please take a look
-    if you don't know how to host Red on a VPS.
+    if you don't know how to host Red.
 
-This is a list of the recommended VPS providers.
+
+| For your instance of Red to stay online it needs to be hosted in a system.
+  This page contains hosting related information and advice for beginners in 
+  topics such as picking a provider.
+
+First, we would like to make something clear:
+
+.. warning::
+    Due to their inability to handle Red's data structure and meet the
+    conditions of being a supported platform; platforms such as Heroku, 
+    Pterodactyl, repl.it, Termux, LXC and alike are **NOT** officially supported. 
+    Docker support found in GitHub is also a work in progress and not ready
+    for daily use. Workarounds for getting Red running on those platforms
+    are imperfect due to Red's nature. And one will not be able to recieve
+    support if an issue occurs when hosting on any of these platforms.
+
+
+------------------------------------
+Hosting on a VPS or Dedicated Server
+------------------------------------
+
+| One can host Red in a VPS running Linux or Windows. Using a Linux VPS is the
```

Comment:
```suggestion
| You can host Red in a VPS running Linux or Windows. Using a Linux VPS is the
```
For ""formal"" writing, you don't want to use "you", but for documentation it's fine

Reply:
I just tend to use "One" more frequently than most, thank you!